src/components/Morpher.tsx

```diff
@@ -1,20 +1,6 @@
 import React, { useState, useEffect } from "react"
-import styled from "@emotion/styled"
 import Link from "./Link"
 
-const NavLink = styled(Link)`
-  text-decoration: none;
-  font-size: 1rem;
-  color: ${(props) => props.theme.colors.text};
-  &:hover {
-    text-decoration: none;
-    color: ${(props) => props.theme.colors.primary};
-  }
-  &.active {
```

Comment:
I couldn't tell what the `.active` class is doing so I couldn't compare, but decided to leave as is as it's not in the scope of this PR to fix this

Reply:
I think that is fine. We were not using it before. This class is related to the active link class that Gatsby allows you to pass. https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/#link-api-surface-area